README.md

```diff
@@ -1,6 +1,130 @@
 # application-aware-quota
 Empower users to customize and manage resource quotas per component in Kubernetes, utilizing the scheduling gates feature.
 
+## Motivation
+In a multi-tenant cluster where multiple users or teams share a Kubernetes cluster with a fixed number of nodes, 
+there is a risk that one team might consume more than its fair share of resources. 
+Resource quotas are a tool for administrators to manage and mitigate this concern.
+
+However, the native ResourceQuotas in Kubernetes are admission-based, which can cause the resource quota 
+admission controller to become a bottleneck for the API Server during evaluation and ResourceQuota status update calls. 
+Additionally, native ResourceQuotas are designed to handle predefined pod resources such as `requests.cpu`, 
+`limits.cpu`, `requests.memory`, `limits.memory`, and more. 
+This design is not easily extendable due to the potential burden on the API server, exacerbating the bottleneck.
+
+Native pod resources are often considered implementation details that can change across different development 
+stages or versions of Kubernetes Operators. For example, consider KubeVirt:
+
+KubeVirt is an operator that enables the deployment and management of virtual machines alongside containerized applications. 
+KubeVirt workloads run inside pods, presenting specific challenges with resource quotas:
+
+1. VM Live Migration: During VM live migration, both source and target pods are required.
+Having both pods simultaneously affects quotas, potentially leading to migration issues and upgrade failures.
```

Comment:
Bear in mind we're aiming this for non-kubevirt users who might not understand the mechanics of live migration. I would rephrase this to something more clear to the general reader:
```suggestion
1. VM Live Migration: Sometimes a VM needs to be evicted from a certain node and reschedule onto another node. However VMs are different than containers in the sense that they carry a large state that cannot be deleted during migrations like with pods which are ephemeral in nature. To achieve that, Kubevirt schedules a new VM instance, transfers the state from the old instance, and only then removes the old instance. Having both VM instances (that run inside pods) simultaneously leads to unfair resource accounting and might potentially lead to migration issues and upgrade failures.
```

Reply:
Done.